### ISSUE TITLE
Setting/getting cookies

### ISSUE THREAD

**Issue by y8** [tongueroo/jets#29]
Hi!

Is it possible to manage cookies from jets? Can't find any mention of cookies or sessions in the code.

By the way, awesome project!

**Comment by tongueroo** [comment]
Good question. Unsure. Will dig into this. Should be possible since cookies work via headers and Jets uses the AWS lambda proxy integration which passes all the headers through. Will test 😄

And thanks for the kind words 🎊

**Comment by y8** [comment]
I've tried  `headers["Set-Cookie"] = "foo=bar"` but it seems like `headers[]=` doesn't affect response headers.

**Comment by tongueroo** [comment]
Awesome 🎊 Thanks for testing and the debugging info. Will dig into adding support. 👍

**Comment by y8** [comment]
I've managed to set headers by explicitly calling `render` with `:headers` argument.

My guess that it happens because headers are set from `@options` ivar. 

https://github.com/tongueroo/jets/blob/d170d64939219a0e1ab9cee34d91fc1b11e516aa/lib/jets/controller/renderers/aws_proxy_renderer.rb#L23

**Comment by tongueroo** [comment]
Nice workaround! Also added support with PR #31 The syntax looks something like this:

```ruby
  def index
    response.headers["Set-Cookie"] = "foo=bar"
    response.set_header("Custom", "MyHeader") # also works
  end
```

Based the syntax on: https://iridakos.com/how-to/2018/03/27/custom-http-headers.html

On slow airplane wifi right now 🛩  Once I get to the hotel will test a little more and it'll be in the next release 😁

Thanks so much for reporting and the debugging info.

**Comment by tongueroo** [comment]
Was able to test with the slow wifi 😁 Using Cloud9 on airplane wifi is pretty trippy.  Released the fix for this in PR #31 as well as your PR #30 X-Runtime improvement in v0.8.14

* [CHANGELOG.md](https://github.com/tongueroo/jets/blob/master/CHANGELOG.md)
* Also started a [Jets FAQ](http://rubyonjets.com/faq/)

Thanks again so much for reporting and helping.

**Comment by tongueroo** [comment]
@y8 FYI Released version 1.1.0 [CHANGELOG](https://github.com/tongueroo/jets/blob/master/CHANGELOG.md). In it, there's an cookies helper that makes it easier to set cookies Docs: http://rubyonjets.com/docs/cookies/
